Makefile

```diff
@@ -371,3 +372,9 @@ clean-output:
 
 # Clean all generated files
 clean: clean-box clean-lstmf clean-output	
+
+# do not search for implicit rules here:
```

Comment:
`Makefile` already uses `.SUFFIXES`, so that looks redundant.

Reply:
> `Makefile` already uses `.SUFFIXES`, so that looks redundant.

It isn't, though. (I've seen it try lexer and VCS rules.)

The manual says .SUFFIXES is for (old-fashioned) suffix rules only (`.c.o` etc), but this protects against builtin (implicit) pattern rules.

Not sure if using `-R` covers that already, though (but the user might not know about that).

Reply:
`make -r` should cover that. Maybe it is sufficient to set `MAKEFLAGS += -r` in the `Makefile`.

Reply:
> `make -r` should cover that. Maybe it is sufficient to set `MAKEFLAGS += -r` in the `Makefile`.

Yes, that does work. (Interestingly, although `make -R` is stronger than `make -r`, `MAKEFLAGS += -R` does _not_ imply `MAKEFLAGS += -rR`. Probably a bug.)  

I'll change the PR accordingly.